react/components/SearchQuery.js

```diff
@@ -6,8 +6,13 @@ import searchMetadataQuery from 'vtex.store-resources/QuerySearchMetadata'
 import facetsQuery from 'vtex.store-resources/QueryFacets'
 
 const DEFAULT_PAGE = 1
-const DEFAULT_SKU_FILTER = 'ALL_AVAILABLE'
-const DEFAULT_FACETS_BEHAVIOR = 'Static'
+
+const DEFAULT_QUERY_VALUES = {
```

Comment:
(nitpick) I don't think there's gonna be any actual problems with this, but the values of this object are mutable, so treating this as a constant might be a bit misleading.

I'd just create different constants for each of its items, instead of grouping them into an object

(this is just a suggestion)